doc-deploy-stable/action.yml

```diff
@@ -86,8 +91,8 @@ runs:
         echo "VERSION=$major.$minor" >> $GITHUB_ENV
 
     - name: "Download the HTML documentation artifact in release/${{ github.ref_name }}"
-      if: env.ACCEPTED_FORMAT == 'true'
-      uses: actions/download-artifact@v3
+      if: ${{ env.ACCEPTED_FORMAT == 'true' && inputs.download-artifacts }}
```

Comment:
Not sure if this will work. Pinging @jorgepiloto

Reply:
Why? I think it should (if you mean the ``if`` syntax)

https://github.com/orgs/community/discussions/25641#discussioncomment-3248571

Reply:
You are right...

It should be without the ``${{ }}``.